alica_tests/src/plans/ValueMappingPlanTestPlan.cpp

```diff
@@ -0,0 +1,24 @@
+#include <alica_tests/plans/ValueMappingPlanTestPlan.h>
+
+namespace alica
+{
+ValueMappingPlanTestPlan::ValueMappingPlanTestPlan(PlanContext& context)
+        : AlicaTestsPlan(context)
+{
+}
+
+void ValueMappingPlanTestPlan::onInit()
+{
+    try {
+        LockedBlackboardRO bb(*getBlackboard());
+        assert(bb.get<int64_t>("inputInt") == -8 && "ValueMappingTestPlan: wrong inputInt value");
```

Comment:
can use the gb to set the error instead of assert i.e. same like in the behaviour test